src/style.css

```diff
@@ -0,0 +1,105 @@
+@import url(..\src\reset.css);
+@import url(..\src\normalize.css);
+@import url(..\src\variables.css);
+@import url(..\src\stars.css);
```

Comment:
Will be better to combine all css files on one `index.css` file, not in file with card styles